lib/features/home/view_model/home_interaction_cubit.dart

```diff
@@ -8,14 +8,15 @@ import 'package:hatspace/singleton/hs_singleton.dart';
 
 part 'home_interaction_state.dart';
 
+enum BottomBarItems { explore, booking, message, profile, adding }
```

Comment:
this belongs to UI scope, and should not be put in the cubit

Reply:
It's fine to be in cubit since it will be used to determine the flow after login success